src/vstart.sh

```diff
@@ -1303,6 +1303,9 @@ EOF
                 fi
             fi
 
+            ceph_adm osd pool create rbd 8 8 replicated
```

Comment:
`ceph_adm osd pool create rbd`

That should be enough. With the pg_autoscaler we don't have to specify pgs anymore.